app/src/config.ts

```diff
@@ -1,22 +1,43 @@
+import { computed, signal } from '@preact/signals'
+import Values from 'values.js'
 import { theme as appTheme } from './hooks/use-theme'
-import { signal } from '@preact/signals'
 
 type Theme = 'light' | 'dark'
 
-const light = signal({
+const tintAndShade = (color: string) => {
```

Comment:
Please use 
```suggestion
function getShades(color: string) {
```

Reply:
Also this code is duplicated here https://github.com/blackmann/stylebender/blob/6f4bf93d2e3aed72c4a932902ac784f923b4f83e/app/src/components/Shades.tsx#L15-L32

Please DRY it